pulp_fedora31/Containerfile

```diff
@@ -1,20 +1,20 @@
 FROM pulp/pulp-ci:latest
 
-ARG PULPCORE_VERSION=3.4.1
-ARG PULP_FILE_VERSION=1.0.1
-ARG PULP_CONTAINER_VERSION=2.0.0b2
-ARG PULP_ANSIBLE_VERSION=0.2.0b13
-ARG PULP_RPM_VERSION=3.4.1
+ARG PULPCORE_VERSION=3.6.0
+ARG PULP_ANSIBLE_VERSION=0.2.0
+ARG PULP_CONTAINER_VERSION=2.0.0
+ARG PULP_FILE_VERSION=1.2.0
 ARG PULP_MAVEN_VERSION=0.1.0
+ARG PULP_PYTHON_VERSION=3.0.0b11
+ARG PULP_RPM_VERSION=3.6.0
 
 RUN pip3 install --upgrade \
   requests \
   pulpcore==${PULPCORE_VERSION} \
-  pulp-file==${PULP_FILE_VERSION} \
-  pulp-container==${PULP_CONTAINER_VERSION} \
   pulp-ansible==${PULP_ANSIBLE_VERSION} \
+  pulp-container==${PULP_CONTAINER_VERSION} \
+  pulp-file==${PULP_FILE_VERSION} \
+  pulp-maven==${PULP_MAVEN_VERSION} \
+  pulp-python==${PULP_PYTHON_VERSION} \
   pulp-rpm==${PULP_RPM_VERSION} \
-  pulp-maven==${PULP_MAVEN_VERSION}
 
-RUN ln /usr/local/lib/python3.7/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_ansible.conf
-RUN ln /usr/local/lib/python3.7/site-packages/pulp_container/app/webserver_snippets/nginx.conf /etc/nginx/pulp/pulp_container.conf
```

Comment:
Those lines are most definitely needed.

Reply:
Hi @mdellweg thanks for your answer. How can I fix this? Were the files renamed?
```
ERROR: Invalid requirement: '/usr/local/lib/python3.7/site-packages/pulp_container/app/webserver_snippets/nginx.conf'
Hint: It looks like a path. File '/usr/local/lib/python3.7/site-packages/pulp_container/app/webserver_snippets/nginx.conf' does not exist.

ERROR: Invalid requirement: '/usr/local/lib/python3.7/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf'
Hint: It looks like a path. File '/usr/local/lib/python3.7/site-packages/pulp_ansible/app/webserver_snippets/nginx.conf' does not exist.
```

Reply:
I don't know. Did the pip install path change?

Their path in the package looks right:
https://github.com/pulp/pulp_container/blob/master/MANIFEST.in#L2

Reply:
fixed. I forgot to remove the backslash on the previous RUN statement...